website/showcase.json

```diff
@@ -1,4 +1,13 @@
 [
+  {
+    "name": "Vizgen Single-Cell MERFISH Spatial Visualizer",
+    "url": "https://vizgen.com/applications/#visual",
```

Comment:
This link seems to goes to a pretty generic/busy page - hard to tell what the demo is here. https://vizgen.com/applications/#visual

Reply:
Hi @ibgreen, yeah we're putting in an anchor tag which should be up by tomorrow. This new tag will direct to the visual which is in the middle of the page.

Reply:
Hey @ibgreen, the anchor tag should be functional now and direct to the visual at the link https://vizgen.com/applications/#visual 

Let us know if you would like us to modify anything.